Dockerfile

```
@@ -73,7 +73,7 @@ COPY --from=builder /usr/lib/python3/dist-packages/valhalla/* /usr/lib/python3/d
 # install all the posix locales that we support
 RUN export DEBIAN_FRONTEND=noninteractive && apt update && \
     apt install -y \
-      libcurl4 libczmq4 libluajit-5.1-2 \
+      libcurl4 libczmq4 libluajit-5.1-2 libgdal32 \
```

I would have thought it'd be:
```suggestion
      libcurl4 libczmq4 libluajit-5.1-2 libgdal-dev \
```

---

or oh I guess it's the runner so only need the lib but why 32? is it a version suffix?

---

ok yeah must be the version. unfortunate naming!

---

Yeah it’s like “libcurl4”, some (to me) strange sort of package versioning